cmd/tf-operator.v1/app/options/options.go

```diff
@@ -51,7 +51,7 @@ func NewServerOption() *ServerOption {
 
 // AddFlags adds flags for a specific CMServer to the specified FlagSet.
 func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
-	fs.StringVar(&s.Kubeconfig, "kubeconfig", "", "The path of kubeconfig file")
+	//fs.StringVar(&s.Kubeconfig, "kubeconfig", "", "The path of kubeconfig file")
```

Comment:
is it temporary?

Reply:
Yeah, I can not find a way to workaround it. User can still use `KUBECONFIG` to specify the path of config files.  This is the only user facing change to tf controller.v1 users. 

Next step is to build universal operator to replace tf controller and hit against all test case. Once conformance test pass. We will remove tf controller.v1